r4/fhirpath/README.md

```diff
@@ -0,0 +1,99 @@
+## Code to show the usage of Fhirpath API
+
+```import ballerina/io;
+import wso2healthcare/healthcare.fhir.r4;
+import ballerina/http;
+
+public function main() returns error? {
+    http:Client fhirPathClient = check new ("Fhirpath_api_running_URL");
+
+    string [] fhirPath = ["Patient.address[1].line[0]","Patient.actve","Patient.address[0].line[0]","Patient.address.use"];
```

Comment:
Shall we add several cases, 
- accessing a single fhir path
- accessing multiple fhir paths

Reply:
Fixed in 28ab50418e064931acc8a5e17b22cbfa773c827a